core/DataAccess/Model.php

```diff
@@ -632,7 +632,7 @@ public function startArchive($invalidation)
         $table = Common::prefixTable('archive_invalidations');
 
         // set archive value to in progress if not set already
-        $statement = Db::query("UPDATE `$table` SET `status` = ? WHERE idinvalidation = ? AND status = ?", [
+        $statement = Db::query("UPDATE `$table` SET `status` = ? AND ts_invalidated = NOW() WHERE idinvalidation = ? AND status = ?", [
```

Comment:
would almost rather need a different field for this like `ts_start`? although maybe not too important. Just thought it might make some troubleshooting bit more difficult but I think it's actually fine as it would only affect invalidations that are in progress

Reply:
this was a quick change, I can add an extra column if needed

Reply:
I have no big preference. Leaving this up to you. Not sure how much work it is. I reckon it may be useful for debugging purposes in the future to understand things better but may not be needed